include/secp256k1.h

```diff
@@ -229,7 +229,7 @@ SECP256K1_API secp256k1_context* secp256k1_context_clone(
  *  be used instead.
  *
  *  Args:   ctx: an existing context to destroy, constructed using
- *               secp256k1_context_create or secp256k1_context_clone
+ *               secp256k1_context_create or secp256k1_context_clone (does nothing if NULL)
```

Comment:
I don't think that we want to guarantee this. As I understand it, we don't define any behavior if you pass NULL, so it's UB (we're just nice currently and do nothing :))